Formula/ocm.rb

```diff
@@ -0,0 +1,29 @@
+class Ocm < Formula
+  desc "CLI for the Red Hat OpenShift Cluster Manager"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift-online/ocm-cli/archive/refs/tags/v0.1.65.tar.gz"
+  sha256 "599e03907783cf468d2652ab91b7e6ee31385797269a6d23c68ad3dbacc404f3"
+  license "Apache-2.0"
+  head "https://github.com/openshift-online/ocm-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/ocm"
+    generate_completions_from_executable(bin/"ocm", "completion")
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/ocm version")
+
+    # Test that completions were generated
+    assert_match(/^# bash completion .* -\*- shell-script -\*-$/, (bash_completion/"ocm").read)
+    assert_match(/^# zsh completion .* -\*- shell-script -\*-$/, (zsh_completion/"_ocm").read)
+    assert_match(/^# fish completion .* -\*- shell-script -\*-$/, (fish_completion/"ocm.fish").read)
+
```

Comment:
I can remove them, but I actually added these in my tap because of a case where the completions generated by `generate_completions_from_executable` didn't actually contain the needed completion info.

Reply:
Updated to remove these tests

Reply:
Then in that case, that would be a wide spread `cobra` issue. :)